### PR TITLE
Finetune & Eval

### PR DESCRIPTION
**Pull request by chrispla**

### CODE REVIEW

**Review by Pliploop — APPROVED** [review]
LGTM, there are a looot of changes due to linting, I don't think it'll be able to merge automatically. Will try to review manually tonight

**Comment by Pliploop** [comment]
@chrispla Good for me if you want to merge :))

**Comment by chrispla** [comment]
@Pliploop woops sorry about that, I resolved conflicts with this but I realized too late I was auto-formatting on save. Everything apart from fine-tuning, eval, datasets/loading and configs should be fine to discard. 

Loss doesn't seem to be going down as fast as expected, I'll have another look before merging.

**Comment by chrispla** [comment]
Loss wasn't going as low as expected, but merging for now to avoid merge conflicts piling up
